### DISCUSSION TITLE
docker部署不成功。

### DISCUSSION THREAD

**Discussion by woshiyigebing** [dataelement/bisheng#432] [Q&A] [answered]
使用docker-compose up -d 进行部署，因为端口被占用，所以只修改了docker-compose.yml中mysql ports: - "3307:3306"，和bisheng/config/config.yaml中的database_url:"mysql+pymysql://root:gAAAAABlp4b4c59FeVGF_OQRVf6NOUIGdxq8246EBD-b0hdK_jVKRs1x4PoAn0A6C5S6IiFKmWn0Nm5eBUWu-7jxcqw6TiVjQA==@mysql:3307/bisheng?charset=utf8mb4" 最后docker-compose up运行成功，但是访问3001端口提示500，后台查看日志显示连接不上mysql，请帮忙看一下是哪里设置有问题吗？

**Comment by yaojin3616** [comment, marked as answer]
可以关注二维码，回复更及时。
因为使用的@mysql 是通过的容器ip进行访问，因此port 需要使用容器内port。如果用3307 需要使用ip访问

**Reply by woshiyigebing** [reply]
感谢回复，问题已解决。
